src/types/TodoContext.ts

```diff
@@ -0,0 +1,10 @@
+import { Todo } from './Todo';
+
+export type TodoContext = {
```

Comment:
```suggestion
export interface TodoContext = {
```
use interface prefer